GitCommands/Git/Executable.cs

```diff
@@ -128,8 +138,22 @@ private void OnProcessExit(object sender, EventArgs eventArgs)
                     {
                         try
                         {
-                            var exitCode = _process.ExitCode;
+                            int exitCode = _process.ExitCode;
                             _logOperation.LogProcessEnd(exitCode);
+
+                            if (_throwOnErrorOutput)
+                            {
+                                string errorOutput = _process.StandardError.ReadToEnd().Trim();
+                                if (exitCode != 0 || errorOutput.Length > 0)
+                                {
+                                    ExternalOperationException ex
+                                        = new(command: _process.StartInfo.FileName, _process.StartInfo.Arguments, _process.StartInfo.WorkingDirectory,
+                                                new Exception($"exit code {exitCode}{Environment.NewLine}{errorOutput}"));
```

Comment:
```suggestion
                                                new Exception($"Exit code: {exitCode}{Environment.NewLine}{errorOutput}"));
```

Reply:
I will move the exit code to the `ExternalOperationException`. The user will not be interested very much in the specific value.